sipa/utils/__init__.py

```diff
@@ -10,6 +10,11 @@
 import socket
 import time
 import dataclasses
+import icalendar
```

Comment:
Please adhere to the import order (just run „optimize imports“ in pycharm) in the following three groups, separated by a blank line:

1.  Built-in Python Modules
2. Third-Party Libraries
3. Application modules (here: Sipa)

Reply:
fixed in a1a8d37